orcestra_book/reports/HALO-20240919a.md

```diff
@@ -0,0 +1,260 @@
+---
+platform: HALO
+flight_id: HALO-20240919a
+nickname: Welcome to Cloud Town; Streets, parks and tower blocks
+takeoff: "2024-09-19 11:00:00Z"
+landing: "2024-09-19 20:00:00Z"
+departure_airport: TBPB
+arrival_airport: TBPB
+crew:
+  - name: Clara Bayley
+    job: PI
+  - name: Sabrina Zechlau
+    job: WALES
+  - name: Raphaela Vogel
+    job: HAMP
+  - name: Helene Glöckner
+    job: Dropsondes
+  - name: Patrizia Schoch
+    job: Smart, VELOX, BACARDI
+  - name: Anna Weber
+    job: SpecMACS
+  - name: Chelsea Nam
+    job: Documentation Scientist
+  - name: Julia Windmiller and Janina Boemeke
+    job: Ground contact
+instruments:
+  BACARDI:
+    comment:  None
+    operational: true
+  BAHAMAS:
+    comment: None
+    operational: true
+  Dropsondes:
+    comment: ATC permission revoked at end of 2nd circle; 6 faulty sondes out of 80.
+    operational: true
+  HAMP:
+    comment: None
+    operational: true
+  PI-Camera:
+    comment: None
+    operational: true
+  Smart:
+    comment: None
+    operational: true
+  SpecMACS:
+    comment: None
+    operational: true
+  VELOX:
+    comment: None
+    operational: true
+  WALES:
+    comment: None
+    operational: true
+categories: [ec_under, ec_track, c_north, c_mid, pace_under, meteor, curtain, radar]
+orphan: true
+---
+
+{logo}`PERCUSION`
+
+# {front}`flight_id`
+
+```{badges}
+```
+
+## Crew
+
+```{crew-list}
+```
+
+## Track
+
+![track](../figures/HALO-20240919a/HALO-20240919a-track_174000UTC_withalltracks.jpg)
+
+Flight path superimposed on the natural color image from NOAA's Geostationary Operational Environmental Satellites (GOES) 16 satellite on 19th September 2024 at 17:40 UTC. The location of the aircraft at the time of the image is shown by the red arrow. The points of closest approach to the EarthCARE (green) and PACE (purple) satellite tracks are marked with red crosses. The points of closest approach to METEOR are marked with red dots. The segment of the path highlighted in orange indicates where the roll maneuver for the radar calibration took place. Dropsondes were dropped throughout the flight, including on the EarthCARE and PACE satellite tracks, inside the 2nd and 5th circles, and on the METEOR overpasses.
+
+## Conditions
+
+<center>
+<video width="590" controls="" poster="https://swift.dkrz.de/v1/dkrz_948e7d4bbfbb445fbff5315fc433e36a/ORCESTRA/static/HALO-20240919a-thumb.png">
+  <source src="https://swift.dkrz.de/v1/dkrz_948e7d4bbfbb445fbff5315fc433e36a/ORCESTRA/static/HALO-20240919a.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+</center>
+
+_Flight path superimposed on the color image from NOAA's Geostationary Operational Environmental Satellites (GOES) 16 satellite on 19th September 2024. The green dots mark sucessful dropsonde launches. Credit: Lukas Kluft._
```

Comment:
(see here)

Reply:
Technically, the layer is called "GOES-East ABI GeoColor", but our shortcut is just "visible" 😉

For the dropsondes, it is their actual location at a given time. You can actually see them move a bit for about 30 minutes. The drift is just so small that it looks stationary. But you can see that they all stop slightly west of the track due to the mean wind.

Reply:
Great thanks, I've updated the figure caption.

P.s. Oh yes sorry I'd forgotten you told me that about the dropsondes - they're even cooler than I thought :)